dynomitemanager/src/main/resources/dynomitemanager.properties

```diff
@@ -92,8 +92,14 @@ dm.dynomite.storage.preconnect=true
 # ENV: DM_DYNOMITE_MULTI_DC
 dm.dynomite.multi.dc=true
 
-#florida.dyno.mbuf.size=16384
-#florida.dyno.allocated.messages=200000
+# Dynomite memory buffer (mbuf) size in bytes
+# OLD: florida.dyno.mbuf.size=16384
+# ENV: DM_DYNOMITE_MBUF_SIZE
+dm.dynomite.mbuf.size=16384
```

Comment:
@shailesh33 We won't know unless we get complaints on GH issues.

However, I've been doing a good job of documenting the old vs. new property names in the [dynomitemanager.properties](https://github.com/DynomiteDB/dynomite-manager/blob/archaius_20161117/dynomitemanager/src/main/resources/dynomitemanager.properties) file.

In the properties file I have 3 lines for each property:

- OLD: the old property
- ENV: the environment variable used to override the FP
- then the new property